src/index.scss

```diff
@@ -27,8 +27,12 @@ h1 {
   z-index: -1;
 }
 
-
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+.cta-button { 
```

Comment:
Thanks for moving this, makes way more sense here and I bet we'll use it a ton more.